gix-index/src/decode/mod.rs

```diff
@@ -96,7 +96,8 @@ impl State {
                     let entries_res = match index_offsets_table {
                         Some(entry_offsets) => {
                             let chunk_size = (entry_offsets.len() as f32 / num_threads as f32).ceil() as usize;
-                            let num_chunks = entry_offsets.chunks(chunk_size).count();
+                            // Lengths of the chunks may vary
```

Comment:
If a test failed when trying `chunks_exact()` here, this comment can be removed. That would be my expectation.

Reply:
I've meant to comment that this code handles chunks of varying lengths by design, so that next person wondering if this should be chunks_exact instead won't need to wonder.